JSDemos/Demos/Charts/Colorization/React/data.ts

```diff
@@ -0,0 +1,98 @@
+import { Properties as TreeMapPropType } from 'devextreme/viz/tree_map';
+
+type typeOptionsType = TreeMapPropType['colorizer'];
+
+export const colorizationOptions: { name: string, options: typeOptionsType }[] = [{
+  name: 'Discrete',
+  options: {
+    type: 'discrete',
+    palette: ['Harmony Light'],
```

Comment:
palette can has array type only in cases when you want pass array of colors. when you pass named palette, you can't use array(it leads to wrong behavior)